Src/Scripts/Install-IronPython.ps1

```diff
@@ -0,0 +1,104 @@
+
+.DESCRIPTION
+    This script facilitates installation of IronPython on .NET 6 binaries from a zip file. The zip file is assumed to have content as published on the IronPython's download page. The zip file is produced by IronPython's "package" build target.
+
+.EXAMPLE
+    PS>./make
+    PS>./make package
+    PS>./Src/Scripts/Install-IronPython.ps1 env
+
+    These commands should be issued on a Powershell prompt with the current directory set to the project root.
+    The project is first built, then packaged, and finally the script uses the zipfile produced during packaging to install IronPython in directory "env"
+
+.EXAMPLE
+
+    PS>Invoke-WebRequest -Uri https://github.com/IronLanguages/ironpython3/releases/download/v3.4.0-beta1/IronPython.3.4.0-beta1.zip -OutFile IronPython.3.4.0-beta1.zip
+    PS>Install-IronPython -Path ~/ipyenv/v3.4.0-beta1 -ZipFile IronPython.3.4.0-beta1.zip -Force
+
+    The official binaries are downloaded from GitHub to the current directory and then the installation proceeds using the downloaded zip file. IronPython is installed into ~/ipyenv/v3.4.0-beta1, overwriting any previous installation in that location.
+    This example assumes that the installation script is in a directory on the search path ($env:PATH).
+#>
+[CmdletBinding()]
+Param(
+    # Target directory to which IronPython is to be installed.
+    [Parameter(Position=0, Mandatory)]
+    [string] $Path,
+
+    # The path to the downloaded zip file with IronPython binaries. If empty, the script will try to grab the package directly produced by the local build.
+    [string] $ZipFile,
+
+    # If the target path exists, it will be wiped clean beforehand.
+    [switch] $Force
+)
+
+$ErrorActionPreference = "Stop"
+
+$defaultVersion = "3.4.0-beta1"
+
+if (-not $ZipFile) {
+    # If zipfile path not given, assume that the script is in the standard location within the source tree
+    # and locate the zip archive in the standard location of the package target.
+    $projectRoot = $PSScriptRoot | Split-Path | Split-Path
+    $ZipFile = Join-Path $projectRoot "Package/Release/Packages/IronPython-$defaultVersion/IronPython.$defaultVersion.zip"
+}
+
+if (Test-Path $Path) {
+    if ($Force) {
+        if ((Resolve-Path $Path).Count -gt 1) {
+            Write-Error "Overwriting of multiple destinations not allowed: $Path"
+        }
+        Remove-Item -Path $Path -Force -Recurse
+    } else {
+        Write-Error "Path already exists: $Path"
+    }
+}
+
+# Unzip archive and move files into place
+$unzipDir = Join-Path $Path "zip"
+
+Expand-Archive -Path $ZipFile -DestinationPath $unzipDir
+Move-Item -Path (Join-Path $unzipDir "Lib") -Destination $Path
+Move-Item -Path (Join-Path $unzipDir "net6.0/*") -Destination $Path -Exclude "*.xml","*.dll.config"
+Remove-Item -Path $unzipDir -Recurse
+
+# Remove garbage files
+Remove-Item -Path (Join-Path $Path "Lib/__pycache__") -Recurse -ErrorAction Ignore
```

Comment:
Hmm, you have `__pycache__` files making their way to the package? Probably an issue that needs to be fixed.

Reply:
Right now it is happening only in my macOS working directory, but if I check it out afresh, `__pycache__` is not being included. A similar thing was happening in my Windows workarea on the day of the release, when a fresh checkout helped. It seems that with use, something is happening to the workarea that `make clean` cannot clean. Not a problem for CI builds, but since I intend to use this script from my regular workarea, I want to automatically skip over this stuff without requiring a fresh checkout.

Reply:
Ok, managed to reproduce the failure. I think there might be a few issues:
- Exclusion glob pattern for `__pycache__` in `StdLib.pyproj` is incorrect.
- Staging folder is not getting cleared. This might cause other junk to make its way into the package...
- `msi\obj` is not getting cleaned? Somehow I managed to hit the same issue you had and zapping this folder "fixed" the issue.

Reply:
Does it mean you are going to address these issues? Or shall I? Or we park it for now and leave the script as is? I see you are making great progress towards 3.6 and I don't want to distract you with my stuff.

Reply:
Glob pattern is a quick fix so I was going to do that one. Didn't have any plans for the other two.

Reply:
> * `msi\obj` is not getting cleaned?

I have noticed before that none of the 17 `obj` dirs are ever being cleaned, which on some occasion caused me problems. They usually contain both `Release` and `Debug` subdirs with often some files directly in `obj`. Thinking about a fix, it is not clear which target should clean them. Maybe a new target `clean-all`?

Reply:
I guess part of the question is how much cleaning is expected of "clean". Like if you create a brand new .NET console program then run `dotnet build` and `dotnet clean` there's still a lot of stuff left over in the `obj` folder. Anyway, if we want to force clean everything maybe it could be called `purge`?

Reply:
I am going to file a separate issue for this.